api/src/main/java/eu/coatrack/api/MetricResult.java

```diff
@@ -0,0 +1,68 @@
+package eu.coatrack.api;
+
+
+public class MetricResult {
+    private String username;
+    private long serviceId;
+    private MetricType type;
+    private long callsPerEntry;
+    private String path;
+    private String requestMethod;
+
+    public MetricResult(String username, long serviceId, MetricType metricType, long numberOfCalls, String path, String requestMethod) {
```

Comment:
Can be removed using `@AllArgsContructor`

Reply:
Also take a look at where else all these Lombok annotation could be used to reduce the code size.